.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- Please put the pull request description above -->
+
+---
+
+Please check all the boxes that apply:
+
+- [ ] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
+- [ ] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
+- [ ] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
```

Comment:
Flip the second and third line? `Signed-off-by` certifies copyright policy but not AI policy.

Reply:
My assumption was that AI policy is implied by copyright policy but makes sense, will do.